### PR TITLE
Updated build.gradle, gradle.properties,gradle-wrapper.properties And Updated UI of Toolbar and Navigation Drawer

### PR DESCRIPTION
**Pull request by codegsaini**
**area:build.gradle**
- Gradle version updated

**area:gradle.properties**
- replaced android.enableR8=true to android.enabledD8=true

**area:gradle\wrapper\gradle-wrapper.properties**
- Updated gradle version

**area:activity_main.xml**
- Updated Navigation Icon for better appearence

**area:drawable**
- added navigation_drawer_icon_tint_selector.xml ( It is a selector file for different icon color of navigation item for selected or unselected state)
- added navigation_drawer_text_color_selector.xml ( It is a selector file for different text color of navigation item for selected or unselected state)
- added navigation_item_selected_bg_selector.xml ( It is a selector file for different background of navigation item for selected or unselected state)
- added navigation_item_selected_bg.xml ( It is a layer-list file to be used by selector file for navigation item background for selected state)

**Screenshot of Navigation Drawer updated UI**

Screenshot of Navigation Drawer updated UI

**Tested on Redmi 6A Api 28**

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -64,6 +64,8 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'junit:junit:4.12'
     implementation 'com.android.support:appcompat-v7:28.0.0'
+    
+
```
Comment (Swati4star) [inline app/build.gradle:68]: why this change?

File: app/src/main/java/swati4star/createpdf/activity/MainActivity.java [hunk 2]
```diff
@@ -127,6 +127,7 @@ protected void onDestroy() {
     protected void onResume() {
         super.onResume();
         ActionBar actionBar = getSupportActionBar();
+        actionBar.setHomeAsUpIndicator(R.drawable.ic_menu);
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/MainActivity.java:130]: Do we need this? The app was working fine as is before

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -5,44 +5,45 @@
     android:id="@+id/drawer_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:fitsSystemWindows="true"
-    tools:openDrawer="start">
+    android:fitsSystemWindows="false">
 
-    <android.support.design.widget.CoordinatorLayout
+    <LinearLayout
```
Comment (Swati4star) [inline app/src/main/res/layout/activity_main.xml:10]: I dont understand the need of changes in this file. 

You are just updating the gradle versions, right?

**Review by Swati4star — CHANGES_REQUESTED** [review]
Travis checks are failing. They should pass, and then only we can merge this PR

File: app/build.gradle (hunk 1 above)
Comment (codegsaini) [reply app/build.gradle:68]: It's by mistake. Actually I tried to add a dependency but removed the dependency and the space left by mistake. I'll remove it.

File: app/src/main/java/swati4star/createpdf/activity/MainActivity.java (hunk 2 above)
Comment (codegsaini) [reply app/src/main/java/swati4star/createpdf/activity/MainActivity.java:130]: Please have a look at updated summary and description.

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (codegsaini) [reply app/src/main/res/layout/activity_main.xml:10]: I forgot to mention that also updated UI. Please have a look at updated summary and description.

**Comment by codegsaini** [comment]
> Travis checks are failing. They should pass, and then only we can merge this PR

Working on it...

File: app/build.gradle (hunk 1 above)
Comment (codegsaini) [reply app/build.gradle]: Removed in commit 6d3fc439494f7458fa10e77e342790a05c0f8b3e

File: licenses/android-googletv-license [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
+601085b94cd77f0b54ff86406957099ebe79c4d6
```
Comment (Swati4star) [inline licenses/android-googletv-license:2]: Hey, the other changes look great!

Can you tell me about these licenses file? I dont understand their purpose.
Also, you made a similar change for travis file also.

**Review by Swati4star — CHANGES_REQUESTED** [review]

File: licenses/android-googletv-license (hunk 4 above)
Comment (Swati4star) [reply licenses/android-googletv-license:2]: Is this change to make SDK accept the licenses?

Can we rathar use this command: https://stackoverflow.com/questions/38096225/automatically-accept-all-sdk-licences

Comment (codegsaini) [reply licenses/android-googletv-license]: Yes, you got it right. This change is to make SDK accept the license.

**What was the problem?**
Travis CI was unable to make SDK auto accept the licenses.

**What this change did?**
In this change I added license files ( in the license folder ) from my local directory. And in .travis.yml file edit, told Travis CI to use given license files if not accepting automatically.

**But why this change and not your mentioned simple command?**
Well, I have gone through various trials including your mentioned one, but none of them worked expect this one.

Comment (codegsaini) [reply licenses/android-googletv-license]: In commit df76b8dafb037070c9adf07c56af4e0fdcc09821, removed unnecessary license files and removed unnecessary commands from travis.yml file.

Comment (codegsaini) [reply licenses/android-googletv-license]: I decided to solve the problem in your mentioned way (because it is simple, short & sweet). Although that command didn't work in the past, this time, with a tiny modification, it worked. :)

Thanks for reminding me that command.

Comment (Swati4star) [reply licenses/android-googletv-license]: Great! This is awesome. Thanks for spending so much time in fixing this :)

**Review by Swati4star — APPROVED** [review]

File: licenses/android-googletv-license (hunk 4 above)
Comment (codegsaini) [reply licenses/android-googletv-license]: Your welcome and thank you for your amazing response. It could never had done without your quick response. 👍

**I am working further to make UI better...**

Happy coding!
